advanced/Scripts/update.sh

```diff
@@ -91,12 +91,37 @@ GitCheckUpdateAvail() {
 
   # Fetch latest changes in this repo
   git fetch --quiet origin
```

Comment:
`if is_repo then git fetch` just to defend against calling this function with a non-repo directory.